tcms/telemetry/api.py

```diff
@@ -247,3 +247,40 @@ def _get_count_for(test_executions):
         .annotate(count=Count("case_id"))
         .order_by("case_id")
     )
+
+
+@http_basic_auth_login_required
+@rpc_method(name="Management.performance")
+def performance_telemetry(query=None):
+
+    if query is None:
+        query = {}
+
+    executions = (
+        TestExecution.objects.filter(**query)
+        # count only execution which are finished
+        .exclude(status__weight=0)
+        .order_by("run_id")
+        .values("assignee_id", "run_id")
```

Comment:
Wouldn't an aggregate query with Count and group by be a better choice here?

And if there isn't a big difference you can also add assignee__username into the mix.

Reply:
the charts looks like this in my setup 
![image](https://user-images.githubusercontent.com/18421997/133892960-e867191a-293e-48cd-9879-355512abad3e.png)

this is for 349 executions. I am not able to find how to tell C3 to render some small part of the chart normally and add a horizontal scroll that leads to the other part. moreover, I found this issue that asks for something similar, which is still open - https://github.com/c3js/c3/issues/1702 there is a workaround in the issue, I will try to do something will that

Reply:
> Wouldn't an aggregate query with Count and group by be a better choice here?

can you clarify more? what should we count by?